testbed/HIL_Firmware/PDU_HIL/PDU_HIL/pduOutputs.c

```diff
@@ -0,0 +1,48 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "driver/twai.h"
+#include "driver/gpio.h"
+#include "pduOutputs.h"
+
+void (*relayPduPtr)(void*) = &relayPduOutputs;
+
+twai_message_t pdu_outputs =
+{
+    .identifier = RELAY_PDU_OUTPUTS,
+    .extd = 1,
+    .data_length_code = 2,
+};
+
+void relayPduOutputs(void * pvParameters)
+{
+    TickType_t xLastWakeTime = xTaskGetTickCount();
+    static uint16_t pdu_gpio_data;
+
+    while(1)
+    {
+        pdu_gpio_data = 0;
+        pdu_gpio_data |= gpio_get_level(POW_AUX) << aux;
```

Comment:
The right side of the << needs to be more specific but I think this will be fixed by a past comment

Reply:
like call this offset or smth